README.md

```diff
@@ -66,7 +66,11 @@ Here is the step in order to build the trace viewer
 
 1. Clone this theia-trace-extension repository
 2. `cd theia-trace-extension`
-3. Now you are ready to build the application: `yarn`
+3. Now you are ready to build the application:
+
+```bash
+yarn && yarn all  # install npm dependencies and build all
```

Comment:
We can continue to use "prepare" like that if we chose. But the trend is going in the other direction. From what I understand, `yarn2` took measures to prevent "prepare" being used like this - if/when we eventually upgrade, I think it will be necessary to at least do `yarn && yarn prepare`. See: https://github.com/yarnpkg/berry/issues/2305#issuecomment-753468942

For now I will revert that change in the form of an extra commit on top, that we can squash or remove, before merging.

Reply:
done

Reply:
I can also update the PR description and commit message, if we want to preserve current "yarn prepare" capabilities.

Reply:
I think for now we should preserve the current "yarn prepare" capabilities. You can open a separate issue tracker and PR for changing the behaviour. I think it would be more transparent for the developers to follow.

Reply:
I have pushed a new version, that preserves backward compatibility of the currently used (AFAIK) root `package.json` "scripts" - let me know what you think.